staff/alex-martinez/precourse/bingo/bingo-v2/index.js

```diff
@@ -0,0 +1,168 @@
+//acumuladores
+var countBingo = false;
+var tiradas = 0;
+var aciertos = 0;
+var linea1 = 0;
+var linea2 = 0;
+var linea3 = 0;
+var cantadoLinea = false;
+
+
+//bombo
+var Bombo = new Array();
+
+function llenarBombo(){
+    //llenarmos del 1 al 99
+    for(var i=1; i<100; i++){
+        Bombo.push(i);
+    }
+    return Bombo;
+}
+
+//funcion para sacar uno de los numeros de dentro del bombo
+//aleatoriamente
+function numBombo(){
+    if(Bombo.length >0){
+        //numero aleatorio entre 1 y 99
+        var nAleatiorio = Math.floor(Math.random()*99)+1;
+    }
+    //variable que almacena el numero aleatorio de la posición del array
+    var numArray = Bombo[nAleatiorio];
+    //se elimina del array del bombo el numero aleatorio obtenido
+    Bombo.splice(nAleatiorio,1);
+
+    return numArray;
+}
+
+var bola;
+
+function sacarBola(){
+    //variable que almacena el numero retornado por la funcion
+    bola = numBombo();
+    //si el numero es indefinido volvemos a tirar()
+    if(bola === undefined){
+        sacarBola();
+    }else{
+        console.log(`¡El ${bola}!`);
```

Comment:
Hi Alex! I would use the + operator to concatenate instead of template strings as we are working on ES5

Reply:
Hi Ramon!, It's true, thanks!